roles/openshift_web_console/tasks/install.yml

```diff
@@ -141,3 +141,5 @@
   when: not create_console_project.changed
 
 - include_tasks: start.yml
+
+- include_tasks: smoke_tests.yml
```

Comment:
Do we want to always install the tests or is this optional?

Reply:
definitelly should be optional.. for now we are installing it by default as part of the WIP